todo_app/classModels.py

```diff
@@ -72,4 +73,26 @@ def recent_done_items(self):
         
         @property
         def older_done_items(self): 
-            return self._olderDone
\ No newline at end of file
+            return self._olderDone
+
+class User(UserMixin):
+    def __init__(self, id, active=True):
+        self.id = id
+        self.role = set()
+        self.role.add('writer')
```

Comment:
The idea is to limit who has writer permissions, e.g.
```suggestion
        self.role.add('reader')
        if id == "Tasnima123":
            self.role.add('writer')
```
You may need to make slightly more changes to keep your tests working